spec/support/factories/votes.rb

```diff
@@ -0,0 +1,18 @@
+FactoryBot.define do
+  factory :vote do
+    value 1
+    voteable { build(:url_post) }
+    voter { build(:oliver) }
+
+    trait :down_vote do
+      value -1
+    end
+
+    trait :up_vote do
+      value 1
+    end
+
+    factory :down_vote, traits: [:down_vote]
+    factory :up_vote, traits: [:up_vote]
```

Comment:
huh, I didn't know this was possible.  Super neat!

Reply:
Cool - but I didn't invent anything here - just copied what is in the docs :) https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#traits